requirements/system-requirements.md

```diff
@@ -29,19 +29,20 @@ This article lists the minimum hardware and software requirements for the instal
 
 The application can be installed on a workstation with __Windows 10, Windows 8 or Windows 7 operating system__, but you will not be able to load new SharePoint farm settings, but only connect to an existing SPDocKit database and open already saved farm settings.
 
-From the version 5.2., SPDocKit installed on a workstation also supports connecting to any __SharePoint 2010, SharePoint 2013, SharePoint 2016 or SharePoint Online__ site and real-time viewing and management of permissions. 
+From the version 5.2., SPDocKit installed on a workstation also supports connecting to any __SharePoint 2010, SharePoint 2013, SharePoint 2016__ site and real-time viewing and management of permissions. 
```

Comment:
From version 5.2., SPDocKit installed on a workstation also supports connecting to any __SharePoint 2010, SharePoint 2013, SharePoint 2016__ site and real-time viewing and management of permissions. [article too many]